bindings/python/ML/include/BipedalLocomotion/bindings/ML/velMANN.h

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file MANN.h
+ * @authors Giulio Romualdi
+ * @copyright 2023 Istituto Italiano di Tecnologia (IIT). This software may be modified and
```

Comment:
```suggestion
 * @file PIMANN.h
 * @authors Evelyn D'Elia 
 * @copyright 2024 Istituto Italiano di Tecnologia (IIT). This software may be modified and
```

Reply:
Done in b08168342887bf55884112e615fa1837fb41287b.